implementations/rust/ockam/ockam/src/profile.rs

```diff
@@ -274,11 +274,11 @@ impl Profile {
                         serde_bare::to_vec(c.data()).map_err(|_| OckamError::BareError)?;
                     let data_hash = vault.sha256(data_binary.as_slice())?;
 
-                    if !vault
+                    if vault
                         .verify(c.self_signature(), c.data().public_key(), &data_hash)
-                        .is_ok()
+                        .is_err()
                     {
-                        false;
```

Comment:
This one requires some eyeballs @SanjoDeundiak @mikelodder7. Previously it was falling through after executing the `false;` empty block. I assume if that validation fails we want to abort? Which is this change

Reply:
This is very nice catch. I'll fix it in my Contact PR (because I moved this code there)